https://gitlab.alpinelinux.org/alpine/aports/-/issues/12295

----
test-canonicalize.c:339: assertion 'strcmp (result1, "/") == 0' failed
Aborted (core dumped)
FAIL test-canonicalize (exit status: 134)
----

diff --git a/gettext-tools/gnulib-tests/test-canonicalize-lgpl.c b/gettext-tools/gnulib-tests/test-canonicalize-lgpl.c
index 1b2ad98..ed901f1 100644
--- a/gettext-tools/gnulib-tests/test-canonicalize-lgpl.c
+++ b/gettext-tools/gnulib-tests/test-canonicalize-lgpl.c
@@ -269,8 +269,8 @@ main (void)
 #endif
     if (same)
       {
-        ASSERT (strcmp (result1, "/") == 0);
-        ASSERT (strcmp (result2, "/") == 0);
+        ASSERT (strcmp (result1, "/") == 0 || strcmp (result1, "//") == 0);
+        ASSERT (strcmp (result2, "/") == 0 || strcmp (result2, "//") == 0);
       }
     else
       {
diff --git a/gettext-tools/gnulib-tests/test-canonicalize.c b/gettext-tools/gnulib-tests/test-canonicalize.c
index 6763a52..8336e67 100644
--- a/gettext-tools/gnulib-tests/test-canonicalize.c
+++ b/gettext-tools/gnulib-tests/test-canonicalize.c
@@ -401,10 +401,10 @@ main (void)
 #endif
     if (same)
       {
-        ASSERT (strcmp (result1, "/") == 0);
-        ASSERT (strcmp (result2, "/") == 0);
-        ASSERT (strcmp (result3, "/") == 0);
-        ASSERT (strcmp (result4, "/") == 0);
+        ASSERT (strcmp (result1, "/") == 0 || strcmp (result1, "//") == 0);
+        ASSERT (strcmp (result2, "/") == 0 || strcmp (result2, "//") == 0);
+        ASSERT (strcmp (result3, "/") == 0 || strcmp (result3, "//") == 0);
+        ASSERT (strcmp (result4, "/") == 0 || strcmp (result4, "//") == 0);
       }
     else
       {
